### PR TITLE
"layout_snake" PR

### PR DESCRIPTION
**Pull request by bitsbymax**
All tests passed

### CODE REVIEW

**Comment by Rocketjet** [comment]
- [DEMO LINK](https://Rocketjet.github.io/layout_stars/)
- [TEST REPORT LINK](https://Rocketjet.github.io/layout_stars/report/html_report/)

**Comment by herkoss** [comment]
Block must take up all available space

Screenshot 2021-08-19 at 18 30 04

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1199px;
```
Comment (herkoss) [inline src/style.css:9]: Block must take up all available space

**Review by herkoss — CHANGES_REQUESTED** [review]
Well done!👍
But it needs some refactoring)
Good luck🍀

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```
Comment (herkoss) [inline src/index.html:7]:
```suggestion
    <meta
      name="viewport"
      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
    >
```

**Review by herkoss — APPROVED** [review]
Great!👍😊

**Pull request closed without merging**